cms/models/pagemodel.py

```diff
@@ -449,6 +449,9 @@ def copy(self, site, parent_node=None, language=None,
         from cms.models import PageContent
         from cms.utils.page import get_available_slug
 
+        if user is None:
```

Comment:
I feel like this would be better as 
```suggestion
        if not user:
```

Because there's not a type check so it could be improperly configured by doing the explicit `None` check.

For example;

```
>>> user = None
>>> if user is None:
...     print("Here")
... 
Here
>>> user = []
>>> if user is None:
...     print("Here")
... 
>>>
```